core/internal/runfiles/uploader.go

```diff
@@ -157,6 +182,87 @@ func (u *uploader) lockForOperation(method string) bool {
 	return true
 }
 
+// Begins watching the given path and uploading when the file changes.
+//
+// Note that this uses native filesystem events, and
```

Comment:
the comment is truncated

Reply:
oh oops, the comment was completely wrong. This is from when I was trying out `fsnotify`. Will remove